recipes/svgwrite/0.1.0/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.1":
+    url: "https://gitlab.com/dvd0101/svgwrite/-/archive/master/svgwrite-master.tar.gz"
```

Comment:
master is not a good reference, it changes for each new release. You have two options:
- Create a tag and refer it on this url (best choice)
- Use the date of that commit as version and set the commit as package for downloading

https://github.com/conan-io/conan-center-index/wiki/FAQ#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
I opted for a tag! TY